channel-jira-cloud/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/descriptor/JiraCloudDistributionUIConfig.java

```diff
@@ -71,12 +71,13 @@ public JiraCloudDistributionUIConfig(Gson gson) {
         ConfigField resolveWorkflow = new TextInputConfigField(JiraCloudDescriptor.KEY_RESOLVE_WORKFLOW_TRANSITION, LABEL_RESOLVE_WORKFLOW_TRANSITION, DESCRIPTION_RESOLVE_WORKFLOW_TRANSITION);
         ConfigField openWorkflow = new TextInputConfigField(JiraCloudDescriptor.KEY_OPEN_WORKFLOW_TRANSITION, LABEL_OPEN_WORKFLOW_TRANSITION, DESCRIPTION_OPEN_WORKFLOW_TRANSITION)
                                        .applyRequiredRelatedField(resolveWorkflow.getKey());
+        ConfigField issueSummary = new TextInputConfigField(JiraCloudDescriptor.KEY_ISSUE_SUMMARY, "", "");
```

Comment:
Label? Description?

Reply:
Oh nvm. This tech debt it confusing...

Reply:
I could just not add the field here as well if this makes it too confusing.